test_results.md

```diff
@@ -0,0 +1,156 @@
+# Testing of automated ODB_SI_ALIGN updates
+
+### Test sign of aimpoint offset
+
+The [aimpoint trending page](http://cxc.cfa.harvard.edu/mta/ASPECT/aimpoint_mon/) shows
+the daily update to the aimpoint offsets for ACIS-S and ACIS-I.  This is computed
+within the `plot_aimpoint.py` module.
+
+The aimpoint offsets specify values which will move the mean observed aimpoint so
+that it matches the POG aimpoints for the ACIS SIs.  (HRC is not adjusted because
+there are no pointing sensitive observations with HRC).
+
+The signs of the values are crucial, but performing an end-to-end test is not practical.
+The following three independent analyses have been done:
+
+- T. Aldcroft: used the aimpoint drift plots in Chapter 4 of the POG, which include arrows
```

Comment:
Do you have a summary of that inspection?  I'm not sure what the take-away is from those plots.